### PR TITLE
Add 16.9 release notes

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
Opening this again to do it against master.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# What's New in 16.9
+
+## Knife Improvements
+
+- The `knife bootstrap` command now properly formats the `trusted_certs_dir` configuration value on Windows hosts. Thanks for this fix [@axelrtgs](https://github.com/axelrtgs)
```
Comment (IanMadd) [inline RELEASE_NOTES.md:7]:
```suggestion
- The `knife bootstrap` command now properly formats the `trusted_certs_dir` configuration value on Windows hosts. Thanks for this fix [@axelrtgs](https://github.com/axelrtgs)!
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+- The `knife bootstrap` command now only specifies the ssh option `-o IdentitiesOnly=yes` if keys are present. Thanks for this fix [@drbrain](https://github.com/drbrain)
```
Comment (IanMadd) [inline RELEASE_NOTES.md:8]:
```suggestion
- The `knife bootstrap` command now only specifies the ssh option `-o IdentitiesOnly=yes` if keys are present. Thanks for this fix [@drbrain](https://github.com/drbrain)!
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+- The `knife status` command with the `-F json` flag no longer fails if cloud nodes have no public IP
```
Comment (IanMadd) [inline RELEASE_NOTES.md:9]:
```suggestion
- The `knife status` command with the `-F json` flag no longer fails if cloud nodes have no public IP.
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+
+The `homebrew_cask` resource has been updated to work with the latest command syntax requirements in the `brew` command. Thanks for reporting this issue [@bcg62](https://github.com/bcg62)!
+
+### locale
+
+The allowed execution time for the `locale-gen` command in the `locale` resource has been extended to 1800 seconds to make sure the Chef Infra Client run doesn't fail before the command completes on slower systems. Thanks for reporting this issue [@janskarvall](https://github.com/janskarvall)!
+
+### plist / macosx_service / osx_profile / macos_userdefaults
+
+Parsing of plist files has been improved in the `plist`, `macosx_service`, `osx_profile` and `macos_userdefaults` resources thanks to updates to the plist gem by [@reitermarkus](https://github.com/reitermarkus) and [@tboyko](https://github.com/tboyko).
```
Comment (IanMadd) [inline RELEASE_NOTES.md:31]:
```suggestion
Parsing of plist files has been improved in the `plist`, `macosx_service`, `osx_profile`, and `macos_userdefaults` resources thanks to updates to the plist gem by [@reitermarkus](https://github.com/reitermarkus) and [@tboyko](https://github.com/tboyko).
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,47 @@
...
+
+The `user` resource on Windows hosts now properly handles `uid` values passed as strings instead of integers. Thanks for reporting this issue [@jaymzh](https://github.com/jaymzh)!
+
+### yum_repostiory
+
+The `yum_repository` resource has been updated with a new `reposdir` property to control the path where the Yum repository configuration files will be written. Thanks for suggesting this [@wildcrazyman](https://github.com/wildcrazyman)!
+
+## Security
+
+- The bundled Nokogiri Ruby gem has been updated to resolve [CVE-2020-26247](https://nvd.nist.gov/vuln/detail/CVE-2020-26247)
```
Comment (IanMadd) [inline RELEASE_NOTES.md:43]:
```suggestion
- The bundled Nokogiri Ruby gem has been updated to resolve [CVE-2020-26247](https://nvd.nist.gov/vuln/detail/CVE-2020-26247).
```

**Pull request merged**